### PR TITLE
replace Google Authenticator with FreeOTP

### LINKED ISSUE #2705: Recommend a FOSS OTP tool
# Feature request

## Description
Google's authenticator is proprietary, and FreeOTP,  which is mentioned once in the UI, strings is no longer maintained, even though it could probably still be functional on current Androids.

Perhaps the **main recommendation** and in-line examples should be of FOSS OTP tools, such as [andOTP](https://github.com/andOTP/andOTP) or [FreeOTP](https://github.com/freeotp/freeotp-android), by using their names in the user's directions, with Google's as a secondary recommendation, if at all.

## User Stories
When advising users in similar situation, I recommend FOSS whenever possible. This is even more relevant in contexts of security and privacy.

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Fixes #2705

Google Authenticator is not Free Software, prefer FreeOTP which is available for Android and iOS and tested to work with SecureDrop. It also simplifies the instructions because the interface is the same on both iOS and Android.

## Testing

* Verify the FreeOTP setup instructions at securedrop/journalist_templates/account_new_two_factor.html and docs/servers_2fa.rst work
* Check the URLs at docs/terminology.rst are working
* Verify the images/manual/screenshots/journalist-admin_new_user_two_factor_totp.png screenshot looks good with the updated strings

## Deployment

N/A 

Although there are changes to the deployment instructions, they only relate to the App generating the two-factor authentication token. They can only change deployment if FreeOTP fails where Google Authenticator succeeds.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

**Comment by dachary** [comment]
special thanks to the [mastodon community](https://mastodon.social/users/Keltounet/statuses/99194973035891386) for testing FreeOTP with iOS

**Comment by kushaldas** [comment]
Using FreeOTP with iOS for almost 3 years (was part of Red Hat experience).

**Comment by dachary** [comment]
Transient network error https://circleci.com/gh/freedomofpress/securedrop/5284?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

<pre>
[Errno 104] Connection reset by peer\n\nStoring debug log for failure in /root/.pip/pip.log
</pre>

**Comment by dachary** [comment]
Transient network error https://travis-ci.org/freedomofpress/securedrop/builds/318496952?utm_source=github_status&utm_medium=notification

<pre>
W: Failed to fetch http://ppa.launchpad.net/git-core/ppa/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
</pre>

**Comment by dachary** [comment]
That's a strange error, never seen that before. https://travis-ci.org/freedomofpress/securedrop/builds/319050666?utm_source=github_status&utm_medium=notification

<pre>
"Setting up hardening-includes (2.5ubuntu2.close failed in file object destructor:

sys.excepthook is missing

lost sys.stderr
</pre>

File: securedrop/Makefile [hunk 1]
```diff
@@ -45,3 +45,14 @@ test-config:
          ctx = {"securedrop_app_gpg_fingerprint": "65A1B5FF195B56353CC63DFFCC40EF1228271441"}; \
          ctx.update(dict((k, {"stdout":v}) for k,v in os.environ.iteritems())); \
          ctx = open("config.py", "w").write(env.get_template("config.py.example").render(ctx))'
+
+.PHONY: update-user-guides
+update-user-guides:
```
Comment (redshiftzero) [inline securedrop/Makefile]: nice! :tada:

**Pull request merged**